src/status_im2/contexts/onboarding/common/overlay/view.cljs

```diff
@@ -61,7 +62,7 @@
       :style          (style/blur-container opacity)}
      [blur/view
       {:blur-amount   @blur-amount
-       :blur-radius   25
+       :blur-radius   (if platform/android? 25 10)
```

Comment:
hmm, curios. How about we put these `blur-radius` in quo2.foundations.blur and have them saved as variables? 🤔 same for 30/20. Would be great if these are common discrepancies between ios and android and then we remove all these individual uses of the platform check

Reply:
Hi @J-Son89, that's a nice suggestion, it will make blur consistent across the app.

But in the current PR, I only focused on blur color and tried to make it the same as Figma (as much as possible). Blur parameters still has arbitrary values (just close enough)
We probably need to do the same with blur where we try different values until get the same (satisfactory) output. Putting these values in separate files will probably be a little misleading and will make other developers use them with confidence (while these are not right/perfect)

We probably need to figure out a system to map, Figma blur values with react-native blur values and then put them in the file, so they can be used across the app (probably will need significant research and time investment). wdyt?

Reply:
Yep, that's some good points. I definitely agree that we would benefit if we can figure out some standardised configurations for blur (provided they actually work for our use cases) - so best to leave as a point of research as you said 👍